src/util/typeGuards.ts

```diff
@@ -2,22 +2,39 @@ import {
   WsFormattedMessage,
   WsMessage24hrMiniTickerRaw,
   WsMessage24hrTickerFormatted,
+  WsMessageAggTradeFormatted, WsMessageFuturesUserDataAccountConfigUpdateEventFormatted,
   WsMessageFuturesUserDataAccountConfigUpdateEventRaw,
+  WsMessageFuturesUserDataAccountUpdateFormatted,
   WsMessageFuturesUserDataAccountUpdateRaw,
   WsMessageFuturesUserDataEventFormatted,
-  WsMessageFuturesUserDataOrderTradeUpdateEventRaw,
+  WsMessageFuturesUserDataListenKeyExpiredFormatted,
+  WsMessageFuturesUserDataMarginCallFormatted,
+  WsMessageFuturesUserDataOrderTradeUpdateEventRaw, WsMessageFuturesUserDataTradeUpdateEventFormatted,
   WsMessageKlineFormatted,
   WsMessageKlineRaw,
   WsMessageMarkPriceUpdateEventFormatted,
+  WsMessageSpotBalanceUpdateFormatted,
+  WsMessageSpotOutboundAccountPositionFormatted,
   WsMessageSpotUserDataEventFormatted,
+  WsMessageSpotUserDataExecutionReportEventFormatted,
+  WsMessageSpotUserDataListStatusEventFormatted,
   WsMessageTradeFormatted,
   WsRawMessage,
   WsUserDataEvents,
 } from '..';
```

Comment:
```suggestion
} from '../types';
```

I'm guessing I did this...wondering if this would be better with a more obvious path? Not really a blocking comment since you didn't introduce this here, just food for thought